Example/panAndScroll/index.js

```diff
@@ -75,7 +78,7 @@ export class TapOrPan extends Component {
 export default class Example extends Component {
   render() {
     return (
-      <ScrollView waitFor={['tap', 'pan']}>
+      <ScrollView waitFor={[tap, pan]}>
```

Comment:
The whole point of this change is so that we can use `waitFor` and related methods in a "contextual" manner. That is it should be possible to instantiate as many instances of `Example` component as we could wish.

Now it isn't possible because tap and pan are global and they would be reused in the second instance of `<TapOrPan>` which would result in crash.

So instead of doing it you should declare these refs as instance variables of the `Example` component. Then you want to pass them to `<TapOrPan>` for example like that:
```
export default class Example extends Component {
  tapRef = React.createRef();
  panRef = React.createRef();
  render() {
    return (
      <ScrollView waitFor={[this.tapRef, this.panRef]}>
        <LoremIpsum words={150} />
        <TapOrPan tapRef={this.tapRef} panRef={this.panRef}/>
        <LoremIpsum words={150} />
      </ScrollView>
    );
  }
}
```

Then use `this.props.panRef` and `this.props.tapRef` in the render method of `TapOrPan`